com.unity.shadergraph/Editor/Data/Util/KeywordUtil.cs

```diff
@@ -79,6 +79,10 @@ public static string ToDeclarationString(this KeywordDescriptor keyword)
                     var enumEntryDefinitions = keyword.entries.Select(x => $"{keyword.referenceName}_{x.referenceName}");
                     string enumEntriesString = string.Join(" ", enumEntryDefinitions);
                     return $"#pragma {definitionString} {enumEntriesString}";
+                case KeywordType.MultiCompile:
+                    var multiCompileEntryDefinitions = keyword.entries.Select(x => $"{x.referenceName}");
+                    string multiCompileEntriesString = string.Join(" ", multiCompileEntryDefinitions);
+                    return $"#pragma {definitionString} {multiCompileEntriesString}";
```

Comment:
This is quite different in latest master, so this won't forward port smoothly (it's not hard, but it's not cherry-pick or copy/paste).  What you probably want to do is add a boolean to KeywordDescriptor (something akin to "useKeywordPrefix"), and handle it that way (so you can omit the '_' when you want to).  If you want to see what the new version of this code looks like, see #3705.